gradle-baseline-java/src/main/groovy/com/palantir/baseline/tasks/CheckImplicitDependenciesTask.java

```diff
@@ -97,6 +92,30 @@ public final void checkImplicitDependencies() {
         }
     }
 
+    private String getSuggestionString(ResolvedArtifact artifact) {
+        String artifactNameString;
```

Comment:
I think this would be an OK use of a ternary operator - seems less gross than the `artifactNameString` thingy

Reply:
done